ansible/environments/mac/.gitignore

```diff
@@ -0,0 +1 @@
+hosts
```

Comment:
Please use the main `.gitignore` for that. We don't want the `.gitignore` hell again.